scripts/pre_commit_linter.py

```diff
@@ -21,6 +21,9 @@
 it will lint all JavaScript files in that directory; otherwise,
 it will only lint files that have been touched in this commit.
 
+This script ignores all filepaths contained within the excludeFiles
+argument in .jscsrc
```

Comment:
Maybe add: 

argument in .jscsrc. Note that, as a side-effect, these filepaths will also prevent Python files in those paths from being linted.